lab_1/main.py

```diff
@@ -4,21 +4,64 @@
 Count frequencies dictionary by the given arbitrary text
 """
 
+# 0: входные данные
+text = input('enter your text: \n')
```

Comment:
Please do not use input - it breaks all checks. Use any sample text for testing purposes.